_pages/en/strategy-alternate-model.md

```diff
@@ -0,0 +1,360 @@
+Each policy statement is a declaration of that decision and received the endorsement of its associated area of governance body (endorsements not yet obtained, see section coherent set of actions).
+
+***Governance and Compliance***
+
+1. IT Product teams have the authority to make changes to their IT products directly in production
+2. IT Product teams publish their product metrics internally to ESDC. Metrics comprises of the following:
+    - Accessibility test results
+    - General Quality test results
+    - Security test results
+    - API consumption statistics
+3. Security Assessment & Authority (SA&A) uses the _XXXAlternate-SA&AXXX methodology_
+4. Accessibility Assessment uses the _XXXAlternate-AccessibilityXXX methodology_
+5. Audit Assessment uses the _XXXAlternate-AuditXXX methodology_
+6. IT Projects do not use business requirements documents
+7. IT Projects start only when the following is true:
+    - Metrics of business outcomes are defined and approved by Business Architecture
+    - The business sponsor commits the availability of, at minimum, one business subject matter experts throughout the IT Project
+    - The business sponsor commits the availability of, at minimum, one policy expert throughout the IT Project
+    - The business sponsor commits the availability of a sample of end users throughout the IT Project
+
+***Finance***
+
+1. IT Projects are under $2.5M
+2. IT Projects are taxed $X% yearly; amount to be distributed amongst the IT products involved in the IT Project
+3. IT Projects do not use cost recovery codes, they fund IT Product cost centres
+4. IT Products use a showback billing model, not a cost recovery model
+
+***Architecture***
+
+1. IT Solutions use loose coupling architectures
+2. IT Products do not depend on other products to run in production
+3. IT Products can test, deploy, and make technical changes without dependencies on other product teams
+4. IT Product teams have the authority to choose their IT Products technical stacks
+5. IT Products expose their functionality and data via APIs
+
+***Product Management***
+
+1. IT Products are mapped to the ESDC Business Capability Model
+2. IT Products have a single Product Owner
+3. IT Products are managed by a Product Manager
+4. The Product Owner may also be the Product Manager
+5. IT Product teams are comprised of 9 members maximum
+6. IT Product teams include a Security Champion member
+7. IT Product teams include a Quality Assurance Champion member
+8. IT Products have their own cost centre codes, funded by the Product Owner or the Project Sponsor
+
+***DevOps***
+
+1. IT Product teams use build automation
+2. IT Product teams use test automation for:
+    - Unit testing
+    - Integration testing
+3. IT Product teams use Continuous Integration
+4. IT Product teams use Application Release Automation
+5. IT Product teams use a decentralized version control system
```

Comment:
Decentralized or at least backed up, I would suppose. 

GcCode isnt decentralized, for example. Though, it probably should be ... if we mean has redundancy